src/operations/clawback.js

```diff
@@ -8,25 +8,30 @@ import { decodeAddressToMuxedAccount } from '../util/decode_encode_muxed_account
  * @alias Operation.clawback
  *
  * @param {object} opts - Options object
- * @param {Asset}  opts.asset - The asset being clawed back.
- * @param {string} opts.amount - The amount of the asset to claw back.
- * @param {string} opts.from - The public key of the (muxed) account to claw back from.
+ * @param {Asset}  opts.asset   - The asset being clawed back.
+ * @param {string} opts.amount  - The amount of the asset to claw back.
+ * @param {string} opts.from    - The public key of the (optionally-muxed)
+ *     account to claw back from.
  *
- * @param {string} [opts.source] - The source account for the operation. Defaults to the transaction's source account.
+ * @param {string} [opts.source] - The source account for the operation.
+ *     Defaults to the transaction's source account.
  *
  * @return {xdr.ClawbackOp}
  *
- * @link https://github.com/stellar/stellar-protocol/blob/master/core/cap-0035.md#clawback-operation
+ * @see https://github.com/stellar/stellar-protocol/blob/master/core/cap-0035.md#clawback-operation
  */
-export function clawback(opts = {}) {
+export function clawback(opts) {
   const attributes = {};
-
   if (!this.isValidAmount(opts.amount)) {
     throw new TypeError(this.constructAmountRequirementsError('amount'));
   }
   attributes.amount = this._toXDRAmount(opts.amount);
   attributes.asset = opts.asset.toXDRObject();
-  attributes.from = decodeAddressToMuxedAccount(opts.from);
+  try {
+    attributes.from = decodeAddressToMuxedAccount(opts.from);
+  } catch (e) {
+    throw new Error('from address is invalid');
+  }
```

Comment:
I'm not following how these changes related to enabling muxed addresses by default. Are these changes required for enabling muxed addresses by default, or some other bug being fixed?

Reply:
This was done to line up how clawback assigns the muxed address with all of the other mux-enabled ops (like payment).